packages/block-editor/src/components/off-canvas-editor/style.scss

```diff
@@ -1,397 +1,15 @@
-.block-editor-list-view-tree {
-	width: 100%;
-	border-collapse: collapse;
+.offcanvas-editor__appender .block-editor-inserter__toggle {
+	background-color: #1e1e1e;
```

Comment:
It seems weird that we need to duplicate these styles. Could they live in the component itself?

Reply:
Maybe as a variant? I'm not sure.

Reply:
Don't they live in the component? The off canvas editor component.

Reply:
@draganescu I'm not fully understanding what you wrote. Are you saying we should leave these CSS changes or move them somewhere? 🙇

Reply:
I think they're fine to live in this experimental thing for now. Keeps things easy to track, and they are custom to this inserter implementation.

Reply:
✅